fire/cli/niv/__init__.py

```diff
@@ -223,39 +224,147 @@ def normaliser_placeringskoordinat(λ: float, φ: float) -> Tuple[float, float]:
 
 # -----------------------------------------------------------------------------
 def skriv_ark(
-    projektnavn: str, resultater: Dict[str, pd.DataFrame], suffix: str = "-resultat"
+    projektnavn: str, nye_faneblade: Dict[str, pd.DataFrame], suffix: str = ""
 ) -> bool:
-    """Skriv resultater til excel-fil"""
+    """Skriv resultater til excel-fil
 
-    filnavn = f"{projektnavn}{suffix}.xlsx"
-    if suffix != "":
-        fire.cli.print(f"Skriver: {tuple(resultater)}")
-        fire.cli.print(f"Til filen '{filnavn}'")
+    Basalt en temmeligt simpel operation, men virkemåden er næsten overskygget af
+    kontroller af at filoperationer gik godt. Det er et vilkår ved interaktioner
+    med filsystemer, så for overblikkets skyld kommer her en kort prosabeskrivelse.
 
-    # Giv brugeren en chance for at lukke et åbent regneark
-    while True:
+    1. Flyt projektnavn.xlsx til projektnavn-ex.xlsx
+    2. Læs dict gamle_faneblade fra projektnavn-ex.xlsx
+    3. Fjern elementer fra gamle_faneblade hvis navnet også findes i nye_faneblade
+    4. Skriv nye_faneblade til projektnavn.xlsx
+    5. Skriv gamle_faneblade til projektnavn.xlsx
```

Comment:
Måske er det forvirrende hvis rækkefølgen på fanebladene ændrer sig fra gang til gang?

Reply:
Jeg har overvejet det. Det er lidt et valg mellem pest og kolera: Hvis de nye bliver sat til sidst vil de i næsten alle tilfælde være usynlige (skjult til højre i fanebladsbjælken). Så jeg tænkte at siden der er en umiddelbar fordel (øjeblikkelig synlighed at det man lige har regnet) valgte jeg den rækkefølge, frem for en efterfølgende opfølgning på "hvor er resultaterne henne". Men det er jo nemt at ændre hvis jeg har gættet fejl af nytteværdien :-)

I øvrigt har jeg kun sat PR som draft fordi jeg endnu ikke har fået testet "start fra en ende fra med nyt projekt" endnu. Det ser ud til at virke fint i øvrigt. Så det er fint med kommmmentarer